### DISCUSSION TITLE
Best resources for Rust and WASM applications to AI?

### DISCUSSION THREAD

**Discussion by deliquentspacexer** [MarsArtistsCommunity/CommunityOverview#8] [Q&A] [answered]
Hi! Just interested in finding resrouces pertaining to WebAssembly and Rust as applied to AI. Thank you!

**Comment by oceanbluesky** [comment, marked as answer]
This is what they were provided, cheers

https://github.com/e-tornike/best-of-ml-rust
https://github.com/rustwasm/awesome-rust-and-webassembly
https://github.com/mbasso/awesome-wasm
https://github.com/rustwasm
https://www.arewelearningyet.com

Also fast.ai tutorials

**Reply by deliquentspacexer** [reply]
Thank you Eric!

**Reply by PhdPIML** [reply]
Here's the Rust course used by Google teams:

https://github.com/google/comprehensive-rust

**Reply by deliquentspacexer** [reply]
Thank you Tao!
